src/sessionDappService.ts

```diff
@@ -345,8 +354,8 @@ export class DappService {
           : "execute_from_outside",
       calldata: CallData.compile({ ...outsideExecution, signature }),
     }
-  } 
-  
+  }
+
   private async compileSessionSignatureFromOutside(
```

Comment:
calls are not used, should be removed?

Reply:
should be used here

```typescript
const guardianSignature = await this.argentBackend.signOutsideTxAndSession(
      // calls,
      sessionRequest,
      accountAddress,
      outsideExecution,
      revision,
      this.chainId,
    )
```

waiting for backend endpoint to check how the body is strucutred

Reply:
i think it won't be needed, calls are already included in the `outsideExecution`, and this will be redundant

Reply:
yep, removed in this pr